shaker/libs/pygit2_utils.py

```diff
@@ -100,3 +112,23 @@ def pygit2_check_credentials():
     message_ok = ("shaker.libs.util:pygit2_check_credentials: No credential problems found. ")
     shaker.libs.logger.Logger().debug(message_ok)
     return True
+
+
+def pygit2_check_ssh_agent():
+    """
+    Check for common pygit2 ssh agent problems
+
+    Return:
+        bool: True if no problems found, False otherwise
+    """
+    agent = paramiko.Agent()
+    keys = agent.get_keys()
+    if len(keys) < 1:
+        return False
+    shaker.libs.logger.Logger().debug("shaker.libs.util:check_pygit2: "
+                                      "Please check that the keys listed contain your github key...")
+    for key in keys:
+        message = ("shaker.libs.util:check_pygit2: "
+                   "Found ssh agent key: %s" % key.get_base64())
+        shaker.libs.logger.Logger().debug(message)
```

Comment:
Why is this using `message` when the message body is directly inside the `debug()` a few lines above?